arm-visualstudio/2014-04-01-preview/swagger/Csm.json

```diff
@@ -0,0 +1,870 @@
+        ],
+        "summary": "Extensions_Create",
+        "description": "Registers the extension with a Visual Studio Team Services account.",
+        "operationId": "extensions_Create",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "An object containing additional information related to the extension request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ExtensionResourcePutRequest"
+            }
+          },
+          {
+            "in": "path",
+            "name": "accountResourceName",
+            "description": "The name of the Visual Studio Team Services account resource.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "extensionResourceName",
+            "description": "The name of the extension.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation succeeded. The extension resource was created for the specified account.",
+            "schema": {
+              "$ref": "#/definitions/ExtensionResourceGetResponse"
+            }
+         }
+        },
+        "x-ms-examples": {
+            "createExtensionResource": {
+                "$ref": "../examples/CreateExtensionResource.json"
+            }
+        }	
+      },
+      "delete": {
+        "tags": [
+          "extensionResource"
+        ],
+        "summary": "Extensions_Delete",
+        "description": "Removes an extension resource registration for a Visual Studio Team Services account.",
+        "operationId": "extensions_Delete",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "in": "path",
+            "name": "accountResourceName",
+            "description": "The name of the Visual Studio Team Services account resource.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "extensionResourceName",
+            "description": "The name of the extension.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation succeeded. The extension resource has been deleted for the specified account."
+         }
```

Comment:
Is Extension delete a long running operation? if so express this operation as LRO refer https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-extensions.md#x-ms-long-running-operation